README.md

```diff
@@ -53,6 +53,7 @@ theme = 'hyper' --  theme is doom and hyper default is hyper
 disable_move    --  defualt is false disable move keymap for hyper
 shortcut_type   --  shorcut type 'letter' or 'number'
 change_to_vcs_root -- default is false,for open file in hyper mru. it will change to the root of vcs
+buffer_name = 'Dashboard' -- the name of the dashboard's buffer. Visible depending on the nvim config
```

Comment:
does this option make sense ? can we use it as default.

Reply:
User does not have to specify it explicitly, "Dashboard" is already the default value. But though this option user can change it, for example, to translate to another language.

Maybe I made it unclear in the help file?